web/translation/translate.en_US.toml

```diff
@@ -312,16 +312,16 @@
 "advancedTemplate" = "Advanced"
 "generalConfigs" = "General Strategy"
 "generalConfigsDesc" = "These options will determine general strategy adjustments."
-"logConfigs" = "Log"
-"logConfigsDesc" = "Logs may affect your server's efficiency. It is recommended to enable it wisely only in case of your needs"
+"logConfigs" = "Log Settings"
+"logConfigsDesc" = "Enabling log may impact your machine's performance. It is recommended to enable it wisely only when necessary."
 "blockConfigs" = "Protection Shield"
 "blockConfigsDesc" = "These options will block traffic based on specific requested protocols and websites."
 "blockCountryConfigs" = "Block Country"
 "blockCountryConfigsDesc" = "These options will block traffic based on the specific requested country."
 "directCountryConfigs" = "Direct Country"
 "directCountryConfigsDesc" = "These options will directly forward traffic based on the specific requested country."
 "ipv4Configs" = "IPv4 Routing"
-"ipv4ConfigsDesc" = "These options will route traffic based on specific requested destination via system's IPv4."
+"ipv4ConfigsDesc" = "These options will route traffic based on specific requested destination via machine's IPv4."
```

Comment:
It seems you like `machine` :))
This word is not a well-known name in IT industry.
But machine could have no IP address. Actually `server` could obtain an IP.